### DISCUSSION TITLE
Information about Criteria

### DISCUSSION THREAD

**Discussion by Nyamort** [symfony/symfony#46524] [Q&A] [answered]
Hello,
I have a table named "document". This table has a one to many relationship with a table named "document_droit_acces". Is It possible to create a criteria that filter documents containing document_droit_acces.droit equal 'CDI' ?
i would like to use this in table 'Contenu' for select documents that containing document_droit_acces.droit equal 'CDI'
![Sans titre](https://user-images.githubusercontent.com/67233336/171197561-49f98e63-446a-4add-81c2-4ba2e16c7d2e.png)
Thanks

**Comment by DylanKas** [comment, marked as answer]
I believe you cannot create a criteria to check a property from on of your relations (ex: `documents.droit`).

However, if you want to filter it in a method inside your Contenus entity you can filter it with something like this:

```
function getDocumentsByDroit(string $droit = 'CDI')
{
     return $this->documents->filter(fn($key, $element) => $element->getDroit() === $droit);
}
```

**Reply by Nyamort** [reply]
Thanks
